### ISSUE TITLE
It's not possible to install version 0.3.0

### ISSUE THREAD

**Issue by michikeiser** [baerengraben/iobroker.swiss-weather-api#15]
Hi

I removed the previous version and installed version 0.3.0. It seems that the adapter doesn't work correctly.

Here is the installation log with debugging enabled:

![image](https://user-images.githubusercontent.com/36990919/88455160-63d9cb00-ce73-11ea-99c5-b4e42dc0b7c3.png)

Even though the adapter has been installed, there is no entry for the instance.  There are no object entries as well.

![image](https://user-images.githubusercontent.com/36990919/88455234-f1b5b600-ce73-11ea-906f-622bb27bce1f.png)

![image](https://user-images.githubusercontent.com/36990919/88455239-02662c00-ce74-11ea-972f-97961c304dca.png)

**Comment by baerengraben** [comment]
Hi
I've done an update today and pushed it to npmjs.com. Tomorrow Version 0.3.0 should be in the latest repo of ioBroker. Can you please do a retest with this version?

**Comment by michikeiser** [comment]
Hello

Thanks for a new version.

It doesn't work. Same behaviour. Sorry.

**Comment by baerengraben** [comment]
Hi
Weird. Not sure if this is an adapter problem or a problem with the ioBroker installation. This is because the installation of the adapter obviously runs without errors, but then no instance appears. Does the configuration overview appear after the installation of the adapter, like this?
![Bildschirmfoto vom 2020-07-30 08-22-29](https://user-images.githubusercontent.com/9572444/88888206-3065ba80-d23e-11ea-9910-5ef077e2cfc4.png)

**Comment by michikeiser** [comment]
Ok, I made some more tests. So if I add an additional instance, it works as expected and the configuration page pops up. But it seems when I install the adapter no instance was created for the first time. I had to add an instance manually, by clicking the "plus" symbol next to the installed adapter. I currently can't put in my secret because the developer page from SRG is down at the moment. I'll try it later. Thank you for your assistance.
